deployment/ccip/changeset/solana/cs_chain_contracts_test.go

```diff
@@ -693,15 +694,15 @@ func TestTokenAdminRegistry(t *testing.T) {
 			require.NoError(t, err)
 
 			tokenAdminRegistryPDA, _, _ := solState.FindTokenAdminRegistryPDA(tokenAddress, state.SolChains[solChain].Router)
-			var tokenAdminRegistryAccount solRouter.TokenAdminRegistry
+			var tokenAdminRegistryAccount solCommon.TokenAdminRegistry
 			err = e.SolChains[solChain].GetAccountDataBorshInto(ctx, tokenAdminRegistryPDA, &tokenAdminRegistryAccount)
 			require.NoError(t, err)
 			require.Equal(t, solana.PublicKey{}, tokenAdminRegistryAccount.Administrator)
 			// pending administrator should be the proposed admin key
 			require.Equal(t, newAdmin, tokenAdminRegistryAccount.PendingAdministrator)
 
 			linkTokenAdminRegistryPDA, _, _ := solState.FindTokenAdminRegistryPDA(linkTokenAddress, state.SolChains[solChain].Router)
-			var linkTokenAdminRegistryAccount solRouter.TokenAdminRegistry
+			var linkTokenAdminRegistryAccount solCommon.TokenAdminRegistry
```

Comment:
@PabloMansanet @toblich couldn't this have just been re-exported from the router?